pages/moods-map.js

```diff
@@ -0,0 +1,95 @@
+import React, { useEffect, useState, useRef } from "react";
+import useSWR from "swr";
+import experienceAnalyser from "@/utils/ExperienceAnalyser";
+import NavButton from "@/components/NavButton/NavButton";
+import { Navigation, Container } from "@/components/Layout/Layout.styled";
+import styled from "styled-components";
+import Circle from "@/components/Circle/Circle";
+import { useRouter } from "next/router";
+import { motion } from "framer-motion";
+import Link from "next/link";
+
+export default function App() {
```

Comment:
you should give the component a more precize name. Having multiple Components named App might lead to trouble